migrations/Version20231130203400.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+class Version20231130203400 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE report ADD uuid VARCHAR(255) NULL');
+        $this->addSql('CREATE UNIQUE INDEX UNIQ_REPORT_UUID ON report (uuid)');
```

Comment:
I don't really have a preference. I just wanted it to have a readably name, but an `idx` in front of it wouldn't hurt :D

Reply:
Actually it should be lower case as well. But more importantly. Since we are using doctrine. You need to define indexes in the entity php file.. For example see how it is done in `User.php`:

```php
#[Table(name: '`user`', uniqueConstraints: [
    new UniqueConstraint(name: 'user_email_idx', columns: ['email']),
    new UniqueConstraint(name: 'user_username_idx', columns: ['username']),
])]
#[Index(columns: ['visibility'], name: 'user_visibility_idx')]
```

After you added a unique contain, you can use symphony doctrine CLI command to generate a migration file for you :).. If you don't do this correctly symfony migration diff will also trigger a difference between the code and the DB in the future.

Reply:
I added it to doctrine via the 'unique' attribute when defining the column. I generated this fipe with the doctrine command and chanhed the index name. Thats all I did 😅

Reply:
The way I did it now causes no diff. When I did it with the `#[Table...]` attribute I it constantly wanted to rename the index even though it had the name that I put into the migration...